dataservice/api/study_file/schemas.py

```diff
@@ -0,0 +1,21 @@
+from marshmallow_sqlalchemy import field_for
+
+from dataservice.api.study_file.models import StudyFile
+from dataservice.api.common.schemas import BaseSchema
+from dataservice.extensions import ma
+
+
+class StudyFileSchema(BaseSchema):
+    study_id = field_for(StudyFile, 'study_id', required=True,
+                         load_only=True)
+
+    class Meta(BaseSchema.Meta):
+        model = StudyFile
+        resource_url = 'api.study_files'
+        collection_url = 'api.study_files_list'
+
+    _links = ma.Hyperlinks({
+        'self': ma.URLFor(Meta.resource_url, kf_id='<kf_id>'),
+        'collection': ma.URLFor(Meta.collection_url),
+        'study_files': ma.URLFor('api.studies', kf_id='<study_id>')
```

Comment:
I think this should just be `study`?

Reply:
i dont know the usecase for having 'study' information in _links but, it is easier to get all the study_files from study as each study_file belong to one study. i am saying this because we can remove this line altogether from _links similar to what we did for investigator and study resources.